Automatic Validation ended with: 
> 2026-04-06T23:29:07.1780660Z ##[error] Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0 Line: 2, Column: 25 File: a55066cb7a7977adf1b416e2e2562c5e016f9e51e35d249ac0bbc8688af73207 Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0 Line: 2, Column: 25 File: c8594ea5b0410713652b68873a27ff63b1733189c89b25245e809ec40643e44c Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0 Line: 2, Column: 25 File: d466386d5e7df0d13576cf5a55e55e12e4257e687880574a806fd28278e594c9
2026-04-06T23:29:07.1781821Z ##[error] Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0 Line: 2, Column: 25 File: a55066cb7a7977adf1b416e2e2562c5e016f9e51e35d249ac0bbc8688af73207 Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0 Line: 2, Column: 25 File: c8594ea5b0410713652b68873a27ff63b1733189c89b25245e809ec40643e44c Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0 Line: 2, Column: 25 File: d466386d5e7df0d13576cf5a55e55e12e4257e687880574a806fd28278e594c9

(Automated response - build 1242.)

--- a/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.installer.yaml
+++ b/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.installer.yaml
@@ -1,0 +1,16 @@
+# Created with winget CLI
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.12.0.schema.json
+
+PackageIdentifier: Arnica.DepsGuard
+PackageVersion: 0.1.19
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+  - RelativeFilePath: depsguard.exe
+    PortableCommandAlias: depsguard
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/arnica/depsguard/releases/download/v0.1.19/depsguard-x86_64-pc-windows-msvc.zip
+    InstallerSha256: 38A6E792CD043FB2CAE18F4F8B6403B65378B34ECE1C3CDD6B967A542849F440
+ManifestType: installer
+ManifestVersion: 1.12.0

--- a/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.locale.en-US.yaml
+++ b/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.locale.en-US.yaml
@@ -1,0 +1,26 @@
+# Created with winget CLI
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.12.0.schema.json
+
+PackageIdentifier: Arnica.DepsGuard
+PackageVersion: 0.1.19
+PackageLocale: en-US
+Publisher: Arnica
+PublisherUrl: https://www.arnica.io
+PublisherSupportUrl: https://github.com/arnica/depsguard/issues
+PackageName: DepsGuard
+PackageUrl: https://depsguard.com
+License: MIT
+LicenseUrl: https://github.com/arnica/depsguard/blob/main/LICENSE
+ShortDescription: Harden package manager configs against supply chain attacks
+Description: DepsGuard hardens your package manager configurations against software supply chain attacks. It scans npm, pnpm, uv, and other package manager lockfiles and configs for security misconfigurations.
+Tags:
+  - cli
+  - security
+  - supply-chain
+  - npm
+  - pnpm
+  - uv
+  - dependency
+  - rust
+ManifestType: defaultLocale
+ManifestVersion: 1.12.0

--- a/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.yaml
+++ b/manifests/a/Arnica/DepsGuard/0.1.19/Arnica.DepsGuard.yaml
@@ -1,0 +1,8 @@
+# Created with winget CLI
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.12.0.schema.json
+
+PackageIdentifier: Arnica.DepsGuard
+PackageVersion: 0.1.19
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.12.0

